src/decision/dMCDA.jl

```diff
@@ -792,3 +799,43 @@ function unguided_site_selection(prefseedsites, prefshadesites, seed_years, shad
 
     return prefseedsites[prefseedsites.>0], prefshadesites[prefshadesites.>0]
 end
+
+
+"""
+    sum_stat_env(env_layer::NamedDimsArray dims::Union{Symbol,Tuple{Symbol,Symbol}}; w=0.5)
+
+Calculates mean over specified dimensions plus half the standard deviation.
+
+# Arguments
+- `env_layer` : Environmental data layer to calculate the mean of.
+- `dims` : Dimensions to aggregate over.
+- `w` : Weighting for std offset to mean.
+
+# Returns
+- Mean + std of input env_layer, where std is weighted by (1-w), mean is weighted by
+w and aggregated over dims.
+"""
+function sum_stat_env(
```

Comment:
This is named `sum_stat_env()`, which is different from its usage elsewhere.

Is there a test for this?

Reply:
This is due to the function name being requested to be changed multiple times. I've fixed it now, it should be summary_stat_env